docs/contributing-to-keyshade/running-things-locally/running-the-web-package.md

```diff
@@ -0,0 +1,52 @@
+---
+description: Get to know how you can run the web package!
+---
+
+## Running the Web Package
+
+The Web Package resides in the `apps/web` directory. It is powered by NextJS and TypeScript. To run the Web Package locally, do the following:
+
+* Build the web app with this command:
+
+```
+turbo run build

```

Comment:
This command, if run from root, it will build all the apps and packages. Rather use `pnpm build:web` this is going to build web specifically, alternatively if you wanna stick to turbo then use the command `turbo run build --filter=web`